Source/DafnyDriver/Commands/ProofComplexityCommand.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.Linq;
+
+namespace Microsoft.Dafny;
+
+public class ProofComplexityCommand : ICommandSpec {
+  public IEnumerable<Option> Options => new Option[] {
+    Iterations,
+    RandomSeed,
+    Format,
+    IsolateAssertions,
+  }.Concat(ICommandSpec.VerificationOptions).
+    Concat(ICommandSpec.CommonOptions);
+
+  static ProofComplexityCommand() {
+    DafnyOptions.RegisterLegacyBinding(Iterations, (o, v) => o.RandomSeedIterations = (int)v);
+    DafnyOptions.RegisterLegacyBinding(RandomSeed, (o, v) => o.RandomSeed = (int)v);
+    DafnyOptions.RegisterLegacyBinding(IsolateAssertions, (o, v) => o.VcsSplitOnEveryAssert = v);
+    DafnyOptions.RegisterLegacyBinding(Format, (o, v) => o.VerificationLoggerConfigs = v);
+  }
+  private static readonly Option<uint> Iterations = new("iterations", () => 10U,
+    $"Attempt to verify each proof n times with n random seeds, each seed derived from the previous one. {RandomSeed.Name} can be used to specify the first seed, which will otherwise be 0.") {
```

Comment:
The NullReferenceException in CI is because you're referencing `RandomSeed` before it's defined below. :) Swapping the ordering of the two options fixed it for me.